app/Http/Requests/MeetingRequest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Enums\MeetingType;
+use App\Enums\ProvinceOrTerritory;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+use Illuminate\Validation\Rules\Enum;
+
+class MeetingRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    public function rules(): array
+    {
+        return [
+            'title.*' => 'nullable|string',
+            'title.en' => 'required_without:title.fr',
+            'title.fr' => 'required_without:title.en',
+            'date' => 'required|date',
+            'start_time' => 'required|date_format:G:i',
+            'end_time' => 'required|date_format:G:i',
+            'timezone' => 'required|timezone',
+            'meeting_types' => 'required|array',
+            'meeting_types.*' => [
+                'nullable',
+                new Enum(MeetingType::class),
+            ],
+            'street_address' => [
+                'nullable',
+                Rule::excludeIf(! in_array('in_person', $this->input('meeting_types', []))),
```

Comment:
For things like 'in_person' that are coming from an enum, you may want to access the value from the Enum directly, in case you ever want to change it. e.g. `MeetingType::InPerson->value`. It would also help to explain/track the relationship. In this case you'd probably also want to assign that to a variable so you wouldn't have to type as much. Like `$inPerson = MeetingType::InPerson->value`.